.gitignore

```diff
@@ -1,4 +1,14 @@
+# Global files
 .DS_Store
-/target
-**/node_modules
-node/bin/*/
+
+# Root directories
```

Comment:
nit: Title Case

Also, this might be better labeled as Build Directories?  Or maybe a Rust section.